downloaders/dumps_downloader.py

```diff
@@ -0,0 +1,76 @@
+import urllib.request
+import tarfile
+import sys
+import os
+from bs4 import BeautifulSoup
+
+
+def show_file_progress(block_num, block_size, total_size):
```

Comment:
[reStructured docstrings](https://www.jetbrains.com/help/pycharm/documenting-source-code.html#) missing here.

Reply:
and please use [type annotation](https://docs.python.org/3/library/typing.html)

Reply:
and please add it to every function

Reply:
Let's use [tqdm hook](https://github.com/tqdm/tqdm/blob/master/examples/tqdm_wget.py#L28~L59)